recipes/simplification/meta.yaml

```
@@ -0,0 +1,60 @@
+{% set name = "simplification" %}
+{% set version = "0.7.10" %}
+{% set rdp_version = "0.13.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/simplification-{{ version }}.tar.gz
+  sha256: 4fbecf738eb36b6fe27c62824eb1c4d4f2700b56375e492fe67c1095747d07d3
+
+build:
+  skip: true  # [py<=37]
+  number: 0
+  script_env:
+    - RDPTAG={{ rdp_version }}
+  missing_dso_whitelist:  # [win]
+    - "*/rdp.dll"  # [win]
+
+requirements:
+  build:
+    - python  # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}  # [build_platform != target_platform]
+    - setuptools >=45
+    - setuptools-scm >=6.2
+    - numpy >=1.11.0
+    - cython >=0.29.0
+    - {{ compiler("c") }}
+    - {{ stdlib("c") }}
+  host:
+    - python
+    - setuptools >=45
+    - setuptools-scm >=6.2
+    - numpy >=1.11.0
```

Please keep this unpinned so the global pin is taken
```suggestion
    - numpy
```